wallet/src/de/schildbach/wallet/WalletApplication.java

```diff
@@ -70,18 +35,54 @@
 import android.os.StrictMode;
 import android.os.SystemClock;
 import android.preference.PreferenceManager;
-import androidx.annotation.StringRes;
-import androidx.multidex.MultiDexApplication;
-import androidx.localbroadcastmanager.content.LocalBroadcastManager;
 import android.text.format.DateUtils;
 import android.widget.Toast;
+
+import androidx.annotation.StringRes;
+import androidx.localbroadcastmanager.content.LocalBroadcastManager;
+import androidx.multidex.MultiDexApplication;
+
+import com.google.common.base.Stopwatch;
+import com.jakewharton.processphoenix.ProcessPhoenix;
+
+import org.bitcoinj.core.CoinDefinition;
+import org.bitcoinj.core.Transaction;
+import org.bitcoinj.core.VerificationException;
+import org.bitcoinj.core.VersionMessage;
+import org.bitcoinj.crypto.LinuxSecureRandom;
+import org.bitcoinj.crypto.MnemonicCode;
+import org.bitcoinj.utils.Threading;
+import org.bitcoinj.wallet.Protos;
+import org.bitcoinj.wallet.UnreadableWalletException;
+import org.bitcoinj.wallet.Wallet;
+import org.bitcoinj.wallet.WalletProtobufSerializer;
+import org.dash.wallet.common.Configuration;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.util.concurrent.TimeUnit;
+
```

Comment:
Why does android studio reorder the imports?

Reply:
I used to use `Ctrl+Alt+L` shortcut very often when writing the code. This shortcut automatically reformats the code based on rules defined in Android Studio, it can be also configured to reorganize imports so that they were consistent in the whole project.

Unfortunately, the Bitcoin Wallet was using different formatting rules originally and when Android Studio reorganizes imports (by removing unused entries and so one) it also reorders entries.

Before the redesign, I was avoiding this to happen by formatting only the edited lines (Android Studio can be configured that way by `Ctrl+Alt+Shift+L` shortcut) but in the meantime, I understood it causes a mess in the files, but on the other hand, keeps the PRs easier to review (by the lack unneeded reformats)... 

However, since redesign requires so many changes and adds so many new files, it doesn't make sense to be consistent with the original Bitcoin Wallet formatting rules anymore and it is a good occasion to take the full advantage of auto-formatting ;)

There are still few files left that were not yet reformatted and `WalletApplication.java` was one of them. I hope you don't mind I didn't discuss this with you before...

Reply:
That is fine.